### ISSUE TITLE
Apple Watch app resigning

### ISSUE THREAD

**Issue by rodriguezst** [rileytestut/AltStore#229]
I have developed a simple Apple Watch app that I can install on my Watch using Xcode but when trying to install via AltStore it gives an "unknown error".
Does AltStore support Apple Watch apps and extensions resigning?

**Comment by noah978** [comment]
I don't believe that AltStore has the capabilities to install WatchOS apps yet. I'll add this as an enhancement for the future.

**Comment by TheNoim** [comment]
AltStore only needs to sign correct and then install it as normal app. The watch extension normally should get synced. However, I always get an unknown error when signing the ipa.

**Comment by gsaker** [comment]
Would be great if this could get fixed, I'm getting a "Watchkit2.0 app bundle ID is not prefixed by the parents app bundle ID followed by a ." error.
